app/v2/controllers/V2DeparturesController.scala

```diff
@@ -73,7 +85,13 @@ class V2DeparturesControllerImpl @Inject() (
     with VersionedRouting
     with ErrorTranslator
     with TemporaryFiles
-    with ContentTypeRouting {
+    with ContentTypeRouting
+    with HasActionMetrics {
+
+  lazy val sCounter: Counter = counter(s"success-counter")
+  lazy val fCounter: Counter = counter(s"failure-counter")
+
+  lazy val gauge: Gauge[_] = registry.gauge(s"error-ratio", () => ErrorRatioGauge(sCounter, fCounter))
```

Comment:
Looping back to https://github.com/hmrc/grafana-dashboards/pull/2078/files - you're not using this (and that's perfectly fine, as if we used this metric, Grafana would average this over the instances, not the sum of the two counts). 

If we're not using this, we should remove it, along with `ErrorRatioGauge`.

Reply:
Will remove, since we can get the functionality we need from Grafana